src/components/Player/Opsheet/Opsheet.js

```diff
@@ -1,17 +1,55 @@
 import React, { Component } from 'react';
-import List from './List'
-import Toggle from './Toggle'
+import List from './List';
+import ListType from './ListType';
+
+//function to convert to react
+// function collapseToggle () {
+//   let collapsed = true;
+//   const toggle = document.getElementsByClassName('collapse-toggle')
+//   const opsheetContent = document.getElementsByClassName('list-wrapper')
+  
+//   toggle[0].addEventListener('click', function() {
+//     if (collapsed === true) {
+//       opsheetContent[0].style.display = 'block';
+//       collapsed = false
+//     } else {
+//       opsheetContent[0].style.display = 'none';
+//       collapsed = true
+//     }
+//   })
+// }
```

Comment:
Don't worry about leaving comments in stuff when you're specifically learning or practicing. But when you submit code to any sort of team project, don't leave code like this commented out. One nice feature of version control is that you can always rewind if you want to see how the code worked before. And cluttering up other people's workspaces will frustrate them and you can't stop someone from deleting the comment artifacts you left behind, anyway.

If you want to save a piece of code, make a folder for snippets somewhere outside the repo, or make liberal use of https://gist.github.com. If Evernote let you syntax highlight and show line numbers, I'd pay for a subscription, just to keep my code snippets in the cloud.